src/main/g8/src/main/webapp/templates-hidden/default.html

```diff
@@ -1,44 +1,23 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xmlns:lift="http://liftweb.net/">
-  <head>
-    <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
-    <meta name="description" content="" />
-    <meta name="keywords" content="" />
-    <title class="lift:Menu.title">App: </title>
-    <link href="/assets/css/app.css" rel="stylesheet" type="text/css" />
-    <style class="lift:CSS.blueprint"></style>
-    <style class="lift:CSS.fancyType"></style>
-    <script id="jquery" src="/classpath/jquery.js" type="text/javascript"></script>
-    <script id="json" src="/classpath/json.js" type="text/javascript"></script>
-  </head>
-  <body>
-    <div class="container">
-      <div class="column span-12 last ralign" >
-        <h1 class="alt">app</h1>
-      </div>
 
-      <hr>
+<head>
+  <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
+  <meta name="description" content="" />
+  <meta name="keywords" content="" />
+  <title>App: </title>
+  <link href="/assets/css/app.css" rel="stylesheet" type="text/css" />
+  <script id="jquery" src="/classpath/jquery.js" type="text/javascript"></script>
+  <script id="json" src="/classpath/json.js" type="text/javascript"></script>
+</head>
 
-      <div class="column span-6 colborder sidebar">
-        <hr class="space" >
+<body>
+  <div>
```

Comment:
Any reason we redid all the HTML here?

Reply:
Oops! This commit should be reverted! I don't know how this kind of change ended up here! :-o Good catch.

Reply:
I reverted that commit. Now I'll erase my branch to not accidentally screw up things again.

Reply:
@farmdawgnation Now I think I know how that extra commit was there in begin with. I had this repo cloned earlier for my own purposes, and didn't realize it while I cloned already and started to make changes assuming it was a clean clone. I'm sorry! But I think you can now merge this latest change so the template is reverted as it was originally.